src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -78,20 +106,22 @@ private static void setupLogger(Application application) {
 	/**
 	 * Launch the application.
 	 */
-	public static void main(String[] args) {
-		EventQueue.invokeLater(new Runnable() {
-			public void run() {
-				Application app = new Application("2d jobs Visio");
-				DrawerFeature.setupDrawerPlugin(app);
-				setupDefaultDrawerVisibilityManagement(app);
-
-				DriverFeature.setupDriverPlugin(app);
-				setupDrivers(app);
-				setupPresetTests(app);
-				setupLogger(app);
-
-				app.setVisibility(true);
-			}
+	public static void main(final String[] args) {
+		//pkrystian: using Runnable is old-fashioned and considered as bad practise nowadays.
```

Comment:
Old-fashioned but why "bad practice". Can you support your statement?

Reply:
Nowadays, old approach gives warnings in compilation. It's even considered a code-smell by sonarqube.
It's approach from before Java 8.

From Java 8, we can use lambdas that make code cleaner - no need to write down obvious things (example: we can use simply "()" instead of "new Runnable()"). It's just a newer standard, and a good practise is to always be up to date with standards.